rpcs3/rpcs3qt/trophy_manager_dialog.cpp

```diff
@@ -321,7 +321,16 @@ trophy_manager_dialog::trophy_manager_dialog(std::shared_ptr<gui_settings> gui_s
 		m_game_combo->setCurrentText(m_game_table->item(m_game_table->selectedItems().first()->row(), GameColumns::GameName)->text());
 	});
 
-	RepaintUI(true);
+	RepaintUI();
+
+	// Fill trophy list
+	auto loadProgress = new QProgressDialog(
+			"Loading trophy data, please wait...", QString(), 0, 1, this,
+			Qt::Dialog | Qt::WindowTitleHint | Qt::CustomizeWindowHint);
```

Comment:
why double tabs?

Reply:
Some code conventions recommend double indentation in instances like these so that they're easily distinguishable from standard indentation. I'll change it to single tabs if it's the style here.